modules/mobile/app/helpers/mobile/facilities_helper.rb

```diff
@@ -18,13 +18,26 @@ def facilities_service
     end
 
     def address_from_facility(facility)
-      address = facility.address['physical']
+      if facility.type == 'va_health_facility' # for MFS Facilities
```

Comment:
So it seems like this PR really does two things: 
1) fixes the handling of VHAFacility because it isn't formatting correctly. Is it possible to fix this in the model? I looked briefly at this model and it's not clear to me why this one is different than the other sub-classes. IMO this is a bug in how our sub-classes are being handled. 
2) changing how we fetch facility data based on a feature flag. Is there a reason we're feature flagging this and not just fixing the bug?

Reply:
It's a feature flag because we are trying to retrieve the facilities from using VAOS's Mobile Facility Service. The facilities are normally retrieved from Lighthouse, but there are some facilities not found in Lighthouse that can be found in MFS.

Reply:
I looked more closely at the story and see now that the plan is to be able to switch back if the new approach causes more problems. Makes sense. Any idea about why that one facility sub-class is different than the others? I don't see us doing this same type checking elsewhere in the app, so I'm curious if it's truly necessary. Were you encountering issues when using strings instead of symbols?

Reply:
Lighthouse and vaos sometimes used different key names for the values the method is looking for. So I thought the type check was necessary. And yes I was encountering issues when using strings instead of symbols

Reply:
There is an option to symbolize the keys. 

    def address_from_facility(facility)
      if facility.type == 'va_health_facility' # for MFS Facilities
        address = facility.physical_address
        street = address[:line].compact.join(', ')
        zip_code = address[:postal_code]
      else
        address = facility.address['physical']
        street = address.slice('address_1', 'address_2', 'address_3').values.compact.join(', ')
        zip_code = address['zip']
      end
      Mobile::V0::Address.new(
        street: street,
        city: address.symbolize_keys[:city],
        state: address.symbolize_keys[:state],
        zip_code: zip_code
      )
    end

Reply:
I feel as if there is a better fix to this. There are two places where this method is called. To avoid confusion in the future, it would be best for both calls to send the same params and the hash keys can be symbolized.